cumulusci/cli/config.py

```diff
@@ -50,6 +50,24 @@ def _load_keychain(self):
             self.keychain = self.keychain_class(self.project_config, self.keychain_key)
             self.project_config.set_keychain(self.keychain)
 
+    def alert(self, message=None):
+        try:
+            if self.project_config.dev_config__no_alert:
+                return
+        except AttributeError:
+            pass
+        if message is None:
+            message = "We need your attention!"
+        click.echo("\a")
+        try:
+            os.system(
+                """osascript -e 'display notification "{}" with title "{}"'""".format(
+                    message, "CumulusCI"
+                )
+            )
+        except:
```

Comment:
Can we catch a specific exception here, or at least `Exception` so it won't catch KeyboardInterrupt?

Reply:
it turns out os.system can never throw an exception unless its called improperly. and for that and some other reasons, its a bad idea. I'll replace w/ subprocess.

Reply:
> can never throw an exception

wait, isn't that what we want here? what are the other reasons?